### ISSUE TITLE
Save to Location changing between session

### ISSUE THREAD

**Issue by twiz-git** [transmission/transmission#6208]
### What is the issue?

I have Transmission 4.0.4 (24077e3511) installed on Windows, and I set the 'Save to Location' (and Keep incomplete files in) to "D:\Downloads". I also have Transmission installed on a Linux machine, with the Save to Location set to "/home/USERNAME/downloads".  
When changing between the two, the 'Save to Location' on Windows gets changed to the Linux location converted to the Windows format.

Steps to reproduce:
1. Starting from a 'local' session on Windows (Edit -> Change Session -> Start Local Session)
2. Edit -> Preferences -> Save to Location is set to "D:\Downloads"
3. Edit -> Change Session -> Connect to Remote Session
4. Edit -> Preferences -> Save to Location is set to "/home/USERNAME/downloads"
5. Edit -> Change Session -> Start Local Session
6. Edit -> Preferences -> Save to Location mouse over shown as "\home\USERNAME\downloads"

Step 2 - Windows using "D:\Downloads" (correct): [image](https://github.com/transmission/transmission/assets/2575677/65405bf1-0950-4c82-9e8a-81267530b4a3)  
Step 4 - Linux using "/home/USERNAME/downloads" (correct): [image](https://github.com/transmission/transmission/assets/2575677/0b3adb4b-92fc-4be7-a21c-197a3dce63f2)
Step 6 - Windows using "/home/USERNAME/downloads" (broken): [image](https://github.com/transmission/transmission/assets/2575677/812d5f0d-93f2-4a2a-bb5d-e67d51b8d520) This image has two circles, one showing the wrong path, and the other 'Calculating Free Space' (due to an invalid path?)

Windows Explorer showing Linux path structure 'converted' to "C:\home\USERNAME\downloads\\,temp": [image](https://github.com/transmission/transmission/assets/2575677/2cd58abc-5801-430d-8762-06335ed1f665)

### Which application of Transmission?

Qt app on Windows

### Which version of Transmission?

 Client: 4.0.4 (24077e3511)  Server: 2.94 (d8e60ee44f)

**Comment by killemov** [comment]
Apparently you are mixing some local/remote sessions here. My best guess is that only the displaying of that path is wrong and should be fixed. As long as you don't save any configuration it should be harmless. Have you tested leaving the session to remote, quit and restart the app and see if the session is still remote and has the correct path?

**Comment by twiz-git** [comment]
> My best guess is that only the displaying of that path is wrong and should be fixed. As long as you don't save any configuration it should be harmless. 

No. As shown in the picture here: [image](https://github.com/transmission/transmission/assets/2575677/2cd58abc-5801-430d-8762-06335ed1f665) it is writing to `C:\home\USERNAME\downloads\.temp` instead of `D:\Downloads\.temp`

> Have you tested leaving the session to remote, quit and restart the app and see if the session is still remote and has the correct path?

It's not the remote session that has the wrong path., it's the local session using the remote session's path.  
But yes, I have tried setting the correct path and exiting it. On relaunch it works as expected ***until*** I connect to remote again and go back to local (either directly or with an exit between), after which it converts the remote sessions Linux path `/home/USERNAME/downloads` to  `\home\USERNAME\downloads` which then saves files as `C:\home\USERNAME\downloads\`.

**Comment by twiz-git** [comment]
I hate to bump topics, but any word on this?  
Is there more info I can provide?

**Comment by killemov** [comment]
Yes, this definitely is a bug. However, I reckon the team will view this as low priority as it is a niche mix of remote and local sessions and the workaround is closing the application between switching sessions.
